src/layer/visual-source.ts

```diff
@@ -25,7 +25,7 @@ interface VisualSource extends Visual {
 }
 
 interface VisualSourceOptions extends VisualOptions {
-  source: HTMLImageElement | HTMLVideoElement
+  source: HTMLImageElement | HTMLVideoElement | string
```

Comment:
If you revert this line and override `source` in `ImageOptions` and `VideoOptions`, will it still compile?

Updating `ImageOptions` to this:
```ts
interface ImageOptions extends VisualSourceOptions {
  source: HTMLImageElement | string
}
```

If it doesn't compile, it's fine to leave it how it is.